Qt-Secret-GUI/secretworker.cpp

```diff
@@ -9,15 +9,14 @@ void SecretWorker::generateKeys(int rsa)
     emit showKeysOnQml();
 }
 
-void SecretWorker::encryptMessage(QByteArray pubKey, QByteArray message)
+void SecretWorker::encryptMessage(QString encPubKey, QString inputText)
 {
-    qDebug() << "input message: " << message;
-    QRSAEncryption::encodeS(message, pubKey);
+    message = QRSAEncryption::encodeS(inputText.toUtf8(), encPubKey.toUtf8(), QRSAEncryption::Rsa(encPubKey.length() * 4));
```

Comment:
QByteArray::fromHex(encPubKey.toUtf8())
it will be right becouse your key in hex format